p2p/state.py

```diff
@@ -58,7 +58,7 @@ def __init__(self,
         self.peer_pool = peer_pool
         self.root_hash = root_hash
         self.scheduler = StateSync(root_hash, account_db)
-        self._running_peers: Set[ETHPeer] = set()
+        self._msg_queue: asyncio.Queue[Tuple[BasePeer, protocol.Command, protocol._DecodedMsgType]] = asyncio.Queue()  # noqa: E501
```

Comment:
This should probably be moved to the `__init__` of `PeerPoolSubscriber` and done in a `super()` call.  I think it is repeated in 4+ locations.

Reply:
Problem is, `PeerPoolSubscriber` is mainly used in classes that have multiple parents and we already `super()` to the other parent's `__init__()`, so we'd need a separate, explicit call to this one's. I'd prefer to keep this small repetition

Reply:
Thoughts on making it lazily initialize itself within a `property` method?

Reply:
Sounds good, done!